packages/cozy-harvest-lib/src/datacards/ViewerModal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from 'react'
+import Modal from '@material-ui/core/Modal'
```

Comment:
We should not import Modal from m-ui. You should use our CozyDialog from cozy-ui (cc @JF-Cozy )

Reply:
It was used in the FileDataCard component I thought it was intended, I'm updating and checking if any error

Reply:
- [ ] Replace with cozy-ui